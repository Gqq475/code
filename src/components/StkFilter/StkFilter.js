import React from 'react'
import classes from './StkFilter.scss'

type Props = {

};
export class StkFilter extends React.Component {
  props: Props;

  render () {
    return (
      <div className={classes['StkFilter-container']} />
    )
  }
}

export default StkFilter
