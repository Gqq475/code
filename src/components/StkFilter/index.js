import StkFilter from './StkFilter'
export default StkFilter
