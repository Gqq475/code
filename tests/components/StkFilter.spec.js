import React from 'react'
import StkFilter from 'components/StkFilter/StkFilter'

describe('(Component) StkFilter', () => {
  it('render normally', () => {
    const wrapper = shallow(
      <StkFilter />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
