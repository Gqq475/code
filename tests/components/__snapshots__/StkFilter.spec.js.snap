// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) StkFilter render normally 1`] = `
<div
  className="StkFilter-container"
/>
`;
